tests/test_functions.py

```
@@ -294,32 +305,124 @@ def build_image_fixture():
+    pg_port = config['postgresql']['port']
+    pg_dbname = config['postgresql']['dbname']
+    schema = get_test_schema()
+
+    docker_exec_prefix = [
+        'docker', 'exec', '--user', 'postgres',
+        '-e', f'PGOPTIONS=-c search_path={schema},public',
+        'test-green-coding-postgres-container',
+        'psql', '-v', 'ON_ERROR_STOP=1', '-d', pg_dbname, '--port', str(pg_port),
+    ]
+
+    check_ps = subprocess.run(
```

Ruff (0.16.0)

[error] `subprocess` call: check for execution of untrusted input

(S603)